src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -783,9 +783,6 @@
   <data name="ERR_V6SwitchGoverningTypeValueExpected" xml:space="preserve">
     <value>A switch expression or case label must be a bool, char, string, integral, enum, or corresponding nullable type in C# 6 and earlier.</value>
   </data>
-  <data name="ERR_IntegralTypeValueExpected" xml:space="preserve">
```

Comment:
>ERR_IntegralTypeValueExpected [](http://example.com/codeflow?start=14&length=29)

Are there matching members in an error enum? I guess the members should be removed, but the values should be kept reserved (unless the members were not shipped yet).

Reply:
@AlekseyTs 

https://github.com/dotnet/roslyn/blob/a2075fd3ca58143b8bc966a117febfd5de50fe6c/src/Compilers/CSharp/Portable/Errors/ErrorCode.cs#L126-L128